providers/env-var/pom.xml

```diff
@@ -12,6 +12,11 @@
     <artifactId>env-var</artifactId>
     <version>0.0.4</version> <!--x-release-please-version -->
 
+    <properties>
+        <!-- override module name defined in parent ("-" is not allowed) -->
+        <module-name>${groupId}.envvar</module-name>
```

Comment:
+1 for name suggestion

However, another alternative is to use underscore `_` , but this is also fine

Reply:
I think if we'd do that, we'd be breaking the "root package" == module name" rule (unless we renamed the packages).